docs/apk-generator.md

```diff
@@ -1,18 +1,18 @@
-## How apk is generated using the Android apk generator
+# How apk is generated using the Android apk generator
 
 ## **Getting the data from uploaded zip/url**
 
-![Apk Generator](images/androidgenerator.png)
+![Apk Generator](_static/images/androidgenerator.png)
```

Comment:
You sure `_static` is needed here? AFAIK `_static` is only needed when referencing images using embedded html.